tutorials/notebooks/mct_features_notebooks/pytorch/example_pytorch_mobilenet_gptq.ipynb

```diff
@@ -0,0 +1,485 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "f8194007-6ea7-4e00-8931-a37ca2d0dd20",
+   "metadata": {
+    "id": "f8194007-6ea7-4e00-8931-a37ca2d0dd20"
+   },
+   "source": [
+    "# Gradient-Based Post Training Quantization using the Model Compression Toolkit - A Quick-Start Guide\n",
+    "\n",
+    "[Run this tutorial in Google Colab](https://colab.research.google.com/github/sony/model_optimization/blob/main/tutorials/notebooks/mct_features_notebooks/pytorch/example_pytorch_mobilenet_gptq.ipynb)\n",
+    "\n",
+    "## Overview\n",
+    "\n",
+    "This tutorial demonstrates a pre-trained model quantization using the **Model Compression Toolkit (MCT)** with **Gradient-based PTQ (GPTQ)**. \n",
+    "\n",
+    "As we will see, GPTQ stands as an optimization procedure that markedly enhances the performance of models undergoing post-training quantization.\n",
+    "This is achieved through an optimization process applied post-quantization, specifically adjusting the rounding of quantized weights.\n",
+    "\n",
+    "In this example, we quantize the model and evaluate the accuracy before and after quantization.\n",
+    "\n",
+    "## Summary\n",
+    "\n",
+    "In this tutorial we will cover:\n",
+    "\n",
+    "1. Gradient-Based Post-Training Quantization using MCT.\n",
+    "2. Loading and preprocessing ImageNet's validation dataset.\n",
+    "3. Constructing an unlabeled representative dataset.\n",
+    "4. Accuracy evaluation of the floating-point and the quantized models.\n",
+    "\n",
+    "## Setup\n",
+    "\n",
+    "Install and import the relevant packages:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "324685b9-5dcc-4d22-80f4-dec9a93d3324",
+   "metadata": {
+    "id": "324685b9-5dcc-4d22-80f4-dec9a93d3324",
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "!pip install -q torch torchvision\n",
+    "!pip install -q model-compression-toolkit\n",
+    "!pip install -q tqdm"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b3f0acc8-281c-4bca-b0b9-3d7677105f19",
+   "metadata": {
+    "id": "b3f0acc8-281c-4bca-b0b9-3d7677105f19"
+   },
+   "outputs": [],
+   "source": [
+    "import torch\n",
+    "from torch.utils.data import DataLoader\n",
+    "from torchvision.models import mobilenet_v2, MobileNet_V2_Weights\n",
```

Comment:
why do you need to import the weights?
can't the model be imported like in this [tutorial](https://github.com/sony/model_optimization/blob/efe8dc9f9d4cfbc6a1254ebe57726fc55916e20e/tutorials/notebooks/mct_features_notebooks/pytorch/example_pytorch_export.ipynb#L84)?

Reply:
This is the way to load pretrained weights. In the tutorial you linked weights are not loaded, in other tutorials the model is loaded with pretrained=Train which is deprecated in torchvision. https://pytorch.org/vision/stable/models.html#initializing-pre-trained-models

Reply:
Also the weights object contains the transforms (it's an official API)